src/env.rs

```diff
@@ -456,4 +456,84 @@ mod tests {
         writer.put("foo", &Value::I64(1235)).expect("wrote");
         writer.delete_value("foo", &Value::I64(1234)).expect("deleted");
     }
+
+    #[test]
+    fn test_iter() {
+        let root = TempDir::new("test_iter").expect("tempdir");
+        fs::create_dir_all(root.path()).expect("dir created");
+        let k = Rkv::new(root.path()).expect("new succeeded");
+        let sk: Store<&str> = k.create_or_open("sk").expect("opened");
+
+        // An iterator over an empty store returns no values.
+        {
+            let reader = sk.read(&k).unwrap();
+            let mut iter = reader.iter().unwrap();
+            assert!(iter.next().is_none());
+        }
+
+        let mut writer = sk.write(&k).expect("writer");
+        writer.put("foo", &Value::I64(1234)).expect("wrote");
```

Comment:
I would be interested in an addition to this test that includes non-ASCII key names.

Reply:
I've added this in 4e33519, which tests keys with both Latin-1 accented and Chinese characters.